### ISSUE TITLE
Generating Date based Square

### ISSUE THREAD

**Issue by cotton1234** [Galarius/gann-square#4]
Hello,

Thanks for the really useful tool.

Its more of cosmetic change, is it possible to have the date in 1 line instead of wrapping in the square so reading becomes easy.

![image](https://github.com/Galarius/gann-square/assets/11193250/7ea9a8f7-4951-4125-8a44-bab42a192d31)

Thanks

**Comment by Galarius** [comment]
Hi @cotton1234,

thank you for the feedback!

The date should somehow fit the square cell and be readable at the same time. I made a separate branch with some changes that allow to display a date in a single line (I modified the font size and the date format). Personally, I find this less readable, but hopefully it will be helpful: https://github.com/Galarius/gann-square/compare/tmp/example/one-line-date

![Screenshot 2023-06-09 at 23 55 07](https://github.com/Galarius/gann-square/assets/7501085/ef9c31a5-9aee-4d3e-bf10-4aff34d9f0e6)
